parts/kubernetesmasteraddons-calico-daemonset.yaml

```diff
@@ -217,8 +217,10 @@ spec:
       hostNetwork: true
       serviceAccountName: canal
       tolerations:
-        - key: node-role.kubernetes.io/master
-          effect: NoSchedule
+        - key: "node-role.kubernetes.io/master"
+          operator: "Equal"
```

Comment:
Equal is the default operator.

Not sure "true" is the default for value. I just point it out here to make sure, it matches master taint.